packages/modeling/src/primitives/circle.js

```diff
@@ -28,7 +28,7 @@ const circle = (options) => {
   }
   let { center, radius, startAngle, endAngle, segments } = Object.assign({}, defaults, options)
 
-  if (!isGT(radius, 0)) throw new Error('radius must be greater than zero')
+  if (!isGTE(radius, 0)) throw new Error('radius must be greater than zero')
```

Comment:
The message could also be changed to "greater than or equal to".

Similarly in other places too.

Reply:
do agree